library/Rules/Length.php

```diff
@@ -46,6 +66,9 @@ public function __construct($min = null, $max = null, $inclusive = true)
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function validate($input): bool
```

Comment:
There are some methods below that are protected, since the class is final now it doesn't make sense for them to be protected anymore.